engine/apps/alerts/models/alert_receive_channel.py

```diff
@@ -552,7 +555,12 @@ def send_demo_alert(self, payload=None):
         if payload is None:
             payload = self.config.example_payload
 
-        if self.has_alertmanager_payload_structure:
+        # TODO: AMV2: hack to keep demo alert working for integration with legacy alertmanager behaviour.
```

Comment:
Could you provide more clearer instruction what should be done here and when?

Reply:
Actually I can't. We still have GRAFANA integration which might or might not have legacy alertmanager behaviour and that's why there is no clear instruction what to do. I'll remove TODO.